src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -264,7 +264,7 @@ private function autocomplete(OutputInterface $output, Question $question, $inpu
             } elseif ("\177" === $c) { // Backspace Character
                 if (0 === $numMatches && 0 !== $i) {
                     --$i;
-                    $fullChoice = substr($fullChoice, 0, -1);
+                    $fullChoice = mb_substr($fullChoice, 0, -1);
```

Comment:
Using `mb_*` without proving the `$encoding` argument is relying on hidden global state - better not.
Could we instead use the function only when $fullChoice is UTF-8? (using `preg_match('//u', ...)`)?